prelexer.cpp

```diff
@@ -423,6 +423,9 @@ namespace Sass {
     const char* number(const char* src) {
       return sequence< optional<sign>, unsigned_number>(src);
     }
+    const char* sass_number(const char* src) {
+      return sequence< optional<sign>, unsigned_number, optional < identifier > >(src);
```

Comment:
What's the use case for `optional < identifier >` here? Are all identifiers really valid in the context?